src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -275,10 +275,11 @@ export const BaseArtworkFilter: React.FC<
 
           <Spacer mb={2} />
 
-          {enableCreateAlert && savedSearchEntity && (
+          {savedSearchEntity && (
```

Comment:
**suggestion:** what do you think of changing that to 
```suggestion
          {!!savedSearchEntity && (
```
just to make sure that nothing weird will happen. not saying that this won't work in react but I think it is better to explicitly make it to a boolean over a falsy value, what do you think?